### DISCUSSION TITLE
Wallpaper and screensaver.

### DISCUSSION THREAD

**Discussion by bobtb98** [rocksdanister/lively#1104] [Q&A] [answered]
I would like to know if it is possible for me to have a picture from my gallery as a wallpaper and a url as a screensaver? 
They are 2 different image

**Comment by rocksdanister** [comment, marked as answer]
Currently active wallpapers are run as screensaver.

In the future it will be possible to have separate screensavers https://github.com/rocksdanister/lively/issues/587
